src/algebra/group/type_tags.lean

```diff
@@ -245,11 +245,14 @@ add_monoid_hom.to_multiplicative''.symm
 
 /-- If `α` has some multiplicative structure and coerces to a function,
   then `additive α` should also coerce to a function. -/
-instance additive.has_coe_to_fun {α : Type*} [has_coe_to_fun α] : has_coe_to_fun (additive α) := 
+@[simps]
```

Comment:
```suggestion
@[simps coe]
```
Should stop the timeout

Reply:
Hm this still fails
```
def has_coe_to_fun.simps.coe {α : Sort*} [has_coe_to_fun α] : Π a : α, has_coe_to_fun.F a :=
coe_fn

/-- If `α` has some multiplicative structure and coerces to a function,
  then `additive α` should also coerce to a function. -/
@[simps coe]
instance additive.has_coe_to_fun {α : Type*} [has_coe_to_fun α] :
  has_coe_to_fun (additive α) :=
⟨λ a, has_coe_to_fun.F a.to_mul, λ a, coe_fn a.to_mul⟩

/-- If `α` has some additive structure and coerces to a function,
	then `multiplicative α` should also coerce to a function. -/
@[simps coe]
instance multiplicative.has_coe_to_fun {α : Type*} [has_coe_to_fun α] :
	has_coe_to_fun (multiplicative α) :=
⟨λ a, has_coe_to_fun.F a.to_add, λ a, coe_fn a.to_add⟩
```

Reply:
Ah, you're right again. My change prevented `#lint` timing out, but it seems now that the `simp` inside `monoid_hom_congr` fails.

Reply:
If it's any consolation, very similar looking simp lemmas seem to be generated with or without the `has_coe_to_fun.simps.coe` line, but maybe I'm misreading the `set_option trace.simps.verbose true` output

Reply:
`@[simps]` should support coercions to functions as a projection by default. For example, for `equiv` is automatically uses `(e : A -> B)` instead of `e.to_fun`. So probably the default behavior is sufficient.

Reply:
It seems that adding simps at all randomly breaks `category_theory/adjunction/opposites.lean:78:6`. Should I just add no simps at the moment, as currently committed? Even without the simps it seems to do what I want, but if it's bad practice I'll experiment a bit more.

Reply:
Usually coercions do not get a `simp` or `simps` attribute, since they are usually the normal form of an expression.

Reply:
Good to know, I'll leave it as is then